contracts/LlamaPay.sol

```diff
@@ -47,8 +47,9 @@ contract LlamaPay {
     }
 
     function createStream(address to, uint216 amountPerSec) public {
-        bytes32 streamId = getStreamId(msg.sender, to, amountPerSec);
+        require(to != address(this), "no naughty");
```

Comment:
why is this important?

Reply:
won't people just lose money if they create a stream to the contract itself?

Reply:
yes, it could be better referred to as "no suicide", but it has enough unintended side-effects on accounting that may cause trouble outside the context, existing tests suggested a potential rogue employer could deposit and withdraw all funds with streams intact i.e. fool people, break expectations and then claim "code is law", more tests could clarify.

Reply:
although it's likely that they are locked away with no direct harm, should check `withdrawable`, nvm me I'm experimenting :P